Resources/doc/reference/configuration/invalidation.rst

```diff
@@ -20,7 +19,25 @@ routes when some other routes are requested.
                         villain_details:     # e.g., /villain/{id}
                             ignore_extra_params: false    # Defaults to true
 
-.. include:: /includes/enabled.rst
+.. include:: ../../includes/enabled.rst
+
+.. include:: ../../includes/expression-language.rst
+
+Your custom expression functions can then be used in the ``@InvalidateRoute`` :ref:`annotations<invalidateroute>`.
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    services:
+        acme.expression_language:
+            class: Acme\TestBundle\Component\ExpressionLanguage
```

Comment:
lets use the new best practices for examples and put this in AppBundle. maybe `AppBundle\ExpressionLanguage\MyExpressionLanguage` ?